### DISCUSSION TITLE
WintermelonC_Docs/zju/compulsory_courses/ADS/ch2/

### DISCUSSION THREAD

**Discussion by giscus** [WintermelonC/WintermelonC_Docs#3] [General]
# WintermelonC_Docs/zju/compulsory_courses/ADS/ch2/

WinteremelonC 的网页

https://wintermelonc.github.io/WintermelonC_Docs/zju/compulsory_courses/ADS/ch2/

**Comment by WintermelonC** [comment]
PTA 2.2 答案有误，我有时间马上修正一下

**Reply by WintermelonC** [reply]
PTA 2.2 答案已修正
